torch/_functorch/_aot_autograd/autograd_cache.py

```diff
@@ -405,6 +412,26 @@ def wrap_post_compile(
 
         Which we'll handle separately later on, if necessary.
         """
+
+        # Log the output of AOTAutogradCache
+        if aot_config.enable_log:
+            # TODO: maybe also log to aot_graphs_log
+            # Unfortunately aot_graphs_log uses
+            # slightly different formatting though
```

Comment:
@bdhirsch I want to also log to aot_graphs_log here but this format is slightly different. Does it matter if it's slightly less formatted/colored? Or should I reimplement `lazy_format_graph_code`?